tests/unit/test_enhanced_types.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+import unittest
+
+from google.cloud.aiplatform.v1beta1.schema.trainingjob import definition
+ModelType = definition.AutoMlImageClassificationInputs().ModelType
+
+
+class EnhancedTypesTests(unittest.TestCase):
```

Comment:
The other library unit tests are written in the style of plain pytest functions.  Unless that is not feasible for what you are testing here, please follow the same convention.  Also perhaps add another folder under `tests/unit` to house tests that specifically have to do with enhanced types, and add yourself as codeowner of that folder (if tests are allowed to have codeowners).

Reply:
Both done: moved tests, switched to pytest rather than unittest.